### PR TITLE
Ja v3 refresh

### PR DESCRIPTION
**Pull request by tcapelle**
Gemini Flash 2.0 translation
- done

### CODE REVIEW

**Comment by johndmulhausen** [comment]
~Error is: `Error: error building site: process: readAndProcessContent: "/opt/buildhome/repo/content/ja/launch/walkthrough.md:78:14": unclosed shortcode`
Fixing now~

Edit: Fixed

**Review by johndmulhausen — APPROVED** [review]
Approving but waiting for guidance on whether to proceed; my suggestion is we wait for our resident Japanese speakers to sanity check.

File: content/ja/guides/_index.md [hunk 1]
```diff
@@ -1,51 +1,50 @@
...
-3. Discover W&B's flexible and lightweight building block for dataset and model versioning with [Artifacts]({{< relref path="/guides/core/artifacts/" lang="ja" >}}).
-4. Automate hyperparameter search and explore the space of possible models with [Sweeps]({{< relref path="/guides/models/sweeps/" lang="ja" >}}).
-5. Manage the model lifecycle from training to production with [Registry]({{< relref path="/guides/core/registry/" lang="ja" >}}).
-6. Visualize predictions across model versions with our [Data Visualization]({{< relref path="/guides/models/tables/" lang="ja" >}}) guide.
-7. Organize runs, embed and automate visualizations, describe your findings, and share updates with collaborators with [Reports]({{< relref path="/guides/core/reports/" lang="ja" >}}).
+1. W&B の基本的な計算単位である [runs]({{< relref path="/guides/models/track/runs/" lang="ja" >}}) について学びます。
+2. [Experiments]({{< relref path="/guides/models/track/" lang="ja" >}}) を使用して、機械学習の実験を作成および追跡します。
+3. [Artifacts]({{< relref path="/guides/core/artifacts/" lang="ja" >}}) を使用して、データセット と モデル の バージョン管理を行うための、W&B の柔軟で軽量な構成要素を見つけます。
```
Comment (olachinkei) [inline content/ja/guides/_index.md:40]: modify to "Artifacts を使用し、データやモデルのバージョン管理をシンプルかつ効率的に行います。"

File: content/ja/guides/_index.md [hunk 2]
```diff
@@ -1,51 +1,50 @@
...
+4. [Sweeps]({{< relref path="/guides/models/sweeps/" lang="ja" >}}) を使用して、ハイパーパラメーター 探索を自動化し、可能なモデルの空間を探索します。
+5. [Registry]({{< relref path="/guides/core/registry/" lang="ja" >}}) を使用して、トレーニング から プロダクション までの モデル ライフサイクルを管理します。
+6. [Data Visualization]({{< relref path="/guides/models/tables/" lang="ja" >}}) ガイドで、モデル バージョン全体の予測を可視化します。
```
Comment (olachinkei) [inline content/ja/guides/_index.md:43]: modify to 
データビジュアライゼーションガイドで、モデルのバージョンごとの予測結果を比較・可視化して理解を深めましょう。

File: content/ja/guides/_index.md [hunk 3]
```diff
@@ -1,51 +1,50 @@
...
+7. [Reports]({{< relref path="/guides/core/reports/" lang="ja" >}}) を使用して、runs の整理、可視化の埋め込みと自動化、発見事項の記述、および コラボレーター との更新の共有を行います。
```
Comment (olachinkei) [inline content/ja/guides/_index.md:44]: Reportsでランを整理し、ビジュアライゼーションを埋め込み、自分の知見を記述・共有：チームとのコラボレーションや情報共有を効率的に行います。

File: content/ja/guides/models/track/launch.md [hunk 4]
```diff
@@ -1,30 +1,30 @@
...
 menu:
   default:
     identifier: ja-guides-models-track-launch
     parent: experiments
-title: Create an experiment
 weight: 1
 ---
 
-Use the W&B Python SDK to track machine learning experiments. You can then review the results in an interactive dashboard or export your data to Python for programmatic access with the [W&B Public API]({{< relref path="/ref/python/public-api/" lang="ja" >}}).
+W&B Python SDK を使用して、 機械学習 の 実験 を追跡します。次に、インタラクティブな ダッシュボード で 結果 を確認するか、[W&B Public API]({{< relref path="/ref/python/public-api/" lang="ja" >}}) でプログラムから アクセス できるように データ を Python にエクスポートできます。
```
Comment (olachinkei) [inline content/ja/guides/models/track/launch.md:11]: Too many spaces

File: content/ja/guides/models/track/launch.md [hunk 5]
```diff
@@ -54,10 +54,10 @@ with wandb.init(
     ...
 ```
 
-For more information on how to configure an experiment, see [Configure Experiments]({{< relref path="./config.md" lang="ja" >}}).
+実験 の構成方法の詳細については、[実験 の構成]({{< relref path="./config.md" lang="ja" >}}) を参照してください。
```
Comment (olachinkei) [inline content/ja/guides/models/track/launch.md:57]: 実験の設定方法の詳細については、[実験の設定]({{< relref path="./config.md" lang="ja" >}}) を参照してください。

File: content/ja/guides/models/track/launch.md [hunk 6]
```diff
@@ -54,10 +54,10 @@ with wandb.init(
...
 
-### Log metrics inside your training loop
-Call [`run.log()`]({{< relref path="/ref/python/log.md" lang="ja" >}}) to log metrics about each training step such as accuracy and loss.
+### トレーニング ループ 内で メトリクス を ログ 記録
+[`run.log()`]({{< relref path="/ref/python/log.md" lang="ja" >}}) を呼び出して、精度や損失など、各トレーニング ステップ に関する メトリクス を ログ 記録します。
```
Comment (olachinkei) [inline content/ja/guides/models/track/launch.md:60]: [`run.log()`]({{< relref path="/ref/python/log.md" lang="ja" >}}) を呼び出して、精度や損失など、各トレーニング ステップ に関する メトリクス を ログします。

File: content/ja/guides/models/track/launch.md [hunk 7]
```diff
@@ -107,19 +106,18 @@ with wandb.init(
     run.log_artifact("path_to_model.onnx", name="trained-model", type="model")
 ```
 
-## Next steps: Visualize your experiment 
-Use the W&B Dashboard as a central place to organize and visualize results from your machine learning models. With just a few clicks, construct rich, interactive charts like [parallel coordinates plots]({{< relref path="/guides/models/app/features/panels/parallel-coordinates.md" lang="ja" >}}),[ parameter importance analyzes]({{< relref path="/guides/models/app/features/panels/parameter-importance.md" lang="ja" >}}), and [more]({{< relref path="/guides/models/app/features/panels/" lang="ja" >}}).
+## 次の ステップ ： 実験 を視覚化する
```
Comment (olachinkei) [inline content/ja/guides/models/track/launch.md:109]: 次の ステップ ： 実験 を可視化する

File: content/ja/guides/core/registry/_index.md [hunk 8]
```diff
@@ -1,148 +1,138 @@
...
     target_path=f"wandb-registry-{REGISTRY_NAME}/{COLLECTION_NAME}"
 )
 ```
 
-W&B automatically creates a collection for you if the collection you specify in the returned run object's `link_artifact(target_path = "")` method does not exist within the registry you specify.
+返された run オブジェクトの `link_artifact(target_path = "")` メソッドで指定したコレクションが、指定した registry 内に存在しない場合、W&B は自動的にコレクションを作成します。
 
 {{% alert %}}
-The URL that your terminal prints directs you to the project where W&B stores your artifact. 
+ターミナル に表示される URL は、W&B が artifact を保存する プロジェクト に移動します。
```
Comment (olachinkei) [inline content/ja/guides/core/registry/_index.md:92]: ターミナルに表示されるURLは、W&Bがアーティファクトを保存しているプロジェクトページへのリンクです。

File: content/ja/guides/core/registry/configure_registry.md [hunk 9]
```diff
@@ -1,127 +1,122 @@
...
+
+| Teams Registry ロール | 個々の Registry ロール | 継承された Registry ロール |
+| ------ | ------ | ------ |
 | Viewer | Viewer | Viewer |
 | Member | Viewer | Member |
-| Admin  | Viewer | Admin  | 
+| Admin  | Viewer | Admin  |
 
-If there is a conflict, W&B displays the highest level of permissions next to the name of the user.
+競合がある場合、W&B は ユーザー の名前の横に最高の権限レベルを表示します。
```
Comment (olachinkei) [inline content/ja/guides/core/registry/configure_registry.md:109]: 権限が重なった場合、W&B はユーザー名の横に一番強い権限を表示します。

**Comment by johndmulhausen** [comment]
I believe we've abandoned this as the GPT version has the left nav translated and this is quite old.
